packages/terra-tabs/src/common-tabs/_Tabs.jsx

```diff
@@ -409,6 +513,25 @@ class Tabs extends React.Component {
         if (tab.isSelected) {
           isHiddenSelected = true;
         }
+        if (index === this.state.visibleTabData.length - 1 && onSelectAddButton) {
+          hiddenTabs.push(
+            <HiddenTab
+              id={this.addTabId}
+              data-focus-styles-enabled
+              itemKey={this.addTabId}
+              label={ariaLabelAddTab}
+              index={enabledTabsIndex + 1}
+              showIcon
+              icon={<IconAdd />}
```

Comment:
don't we need `a11yLabel` here for Add icon button

Reply:
This icon is sent as an parameter for HiddenTab where text will be appended with the icon .